adb-utils.spec

```diff
@@ -44,6 +44,9 @@ ln -s /opt/adb/add_insecure_registry %{buildroot}%{_bindir}/add_insecure_registr
 %doc LICENSE  README.rst
 
 %changelog
+* Fri Apr 01 2016 kumarpraveen.nitdgp@gmail.com 1.4-1
+- Update to 1.4 with couple of bugfix
```

Comment:
Specfile should not list bug fixes for a release, if there is minor version bump then we list specific bugs which caused it.